views/button.pug

```diff
@@ -10,9 +10,15 @@ block content
     div#button-container
       h2(class="title") Update API Data
       label(for="year", class="inputLabels") Year
+      br
       input#API-year-input(type="number", name="year")
       br
+      br
       button#updateButton(class="update") UPDATE
+      br
+      p This will update Median Housing Cost, Median Rent, LaborForce, LaborForce Participation Rate, Employment and Unemployment.
+      br
+
     div#update-table-container
       h1 Manual Indicator Update
       table#update-table
```

Comment:
Add css rule to this element #update-table{margin: auto} to get the table to center on resize. I fiddled with it on my machine and that got the table to stay centered